bootstrap/kubeadm/api/v1alpha3/kubeadmconfig_types.go

```diff
@@ -109,7 +109,7 @@ type KubeadmConfigStatus struct {
 	// BootstrapData will be a cloud-init script for now.
 	//
 	// Deprecated: Switch to DataSecretName.
-	//
+	// TODO: This needs to be kept to support conversion to v1alpha3 types.
```

Comment:
I'm not sure this TODO is helpful. 

We will drop the whole package at some point (when we don't support v1alpha3 anymore)